pkg/virt-launcher/virtwrap/converter/converter.go

```diff
@@ -328,11 +328,11 @@ func getOptimalBlockIOForDevice(path string) (*api.BlockIO, error) {
 	}
 	defer util.CloseIOAndCheckErr(f, nil)
 
-	logicalSize, err := unix.IoctlGetInt(int(f.Fd()), unix.BLKSSZGET)
+	logicalSize, err := unix.IoctlGetUint32(int(f.Fd()), unix.BLKSSZGET)
```

Comment:
TBH I would have preferred that kubevirt delegates this logic to some tool like `blockdev` (C-based :stuck_out_tongue:) since this is very specific knowledge. With that said, the args to `blockdev` will look very similar, and ofc will run a binary, so, I am not that worried.